event_archive.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: past_events_list
 title: Events Archive
-permalink: /events/archive
+permalink: /event/archive
```

Comment:
I think this is better as /events/ because it doesn't expose a directory at /event/ and it's more logically grouped under /events/ as opposed to its own thing at /event/